src/cpdetect/compiler_detector/heuristics/pe_heuristics.cpp

```diff
@@ -675,6 +675,21 @@ void PeHeuristics::getActiveMarkHeuristics()
 	}
 }
 
+/**
+ * Detection of RLPack packer
+ */
+void PeHeuristics::getRLPackHeuristics()
+{
+	if (search.exactComparison("B800000000600BC07458E8000000005805430000008038E9750361EB35E800000000582500F0FFFF33FF66BB195A6683C33466391875120FB7503C03D0BBE944", toolInfo.epOffset) ||
+		search.exactComparison("57C7C772AFB4DF8D3D5FBA581AFFCF0FACF7F20FBDFEF7C75CDC30270FBAF7330FBBF70FCFBF64A909DB85F681DFAC194648F7DF0FA3F7C7C741BC79A085F7D1", toolInfo.epOffset))
+	{
+		addPacker(DetectionMethod::STRING_SEARCH_H, DetectionStrength::HIGH, "RLPack");
+	}
+
+	// TODO: Need to scan other RLPack variants. We would need a disassembler that normalizes instructions
```

Comment:
Instead of leaving a TODO comment, consider creating [an issue](https://github.com/avast/retdec/issues). It is generally better to have all tasks in a bug tracker than spread over source code in comments.

Reply:
https://github.com/avast/retdec/pull/619